src/main/webapp/WEB-INF/views/allCars.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>Display all cars</title>
+</head>
+<body>
+<h1>All cars</h1>
+<form method="get" action="${pageContext.request.contextPath}/cars/delete" id="delete" name="form">
+    <select size="3" form="delete" id="select" name="id">
+        <c:forEach items="${cars}" var="car">
+            <option value="${car.id}"/>${car.id} ${car.model} ${car.manufacturer.id} ${car.manufacturer.name}
+             ${car.manufacturer.country}
+            <c:forEach items="${car.drivers}" var="driver">${driver.id} ${driver.name} ${driver.licenseNumber}</c:forEach>
+            </option>
+        </c:forEach>
+    </select>
+    <button type="submit">Delete</button>
+</form>
+
+</table>
+<a href="${pageContext.request.contextPath}/index">Main</a>
+<a href="${pageContext.request.contextPath}/cars/create">Add car</a>
+<a href="${pageContext.request.contextPath}/manufacturers/create">Add manufacturer</a>
+<a href="${pageContext.request.contextPath}/drivers/create">Add driver</a>
+<a href="${pageContext.request.contextPath}/cars/add">Add driver to car</a>
```

Comment:
"create" is an example of a bad ending for a url. Fix it everywhere

<img width="471" alt="Screen Shot 2022-08-16 at 5 22 38 PM" src="https://user-images.githubusercontent.com/90692308/184903810-f4c2f77d-d91f-49ae-94c5-6513c1423710.png">

Reply:
done